README.md

```diff
@@ -25,11 +25,23 @@ Tools are compiled for the following architectures:
 
 These can be cross-compiled from any system.
 
+### V3 Changes
+The separate executables (like `datasetIngestor`, `datasetRetriever`...) were combined into one `scicat-cli` executable, with each executable's features available as commands given as the first parameter to this executable. 
+
+These commands bare the same names as the former executables. For example, if you called `./datasetIngestor [flags]` before, now it's `./scicat-cli datasetIngestor [flags]`.
+
+ Furthermore, the use of single hyphen, multi-letter flags is now discontinued (as it does not follow convention). 
```

Comment:
sorry for yet another comment, I think this is not very clear, especially for non DEV users

can you maybe add an example? Thanks